docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -167,7 +167,10 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 50034  | A message provided was too old to bulk delete                                                                                 |
 | 50035  | Invalid form body (returned for both `application/json` and `multipart/form-data` bodies), or invalid `Content-Type` provided |
 | 50036  | An invite was accepted to a guild the application's bot is not in                                                             |
-| 50041  | Invalid API version provided                                                                                                  |
+| 50041  | Invalid API version provided                
+|
```

Comment:
I don't think this border and the one on 173 should be on new lines

Reply:
Thanks, fixed